src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -1,13 +1,13 @@
 ---
 title: Migrate your users to the New Relic One user model 
-metaDescription: "For New Relic accounts with users on our original user model: how to migrate your users to our new model."
+metaDescription: "For New Relic organizations with users on our original user model: how to migrate your users to our new model."
 ---
 
-Starting April 12, 2021, we're allowing some customers who have users on our original user model to self-serve and migrate those users to be on the New Relic One user model.
+Starting April 12 of 2021, we're allowing some customers who have users on our original user model to self-serve and migrate those users to be on the New Relic One user model.
```

Comment:
Self-serve is hyphenated as a noun, but as a verb I believe Chicago manual drops the hyphen.

Reply:
funny enough, I was just making a minor edit to retrigger the build. so sometimes when you see me making trivial edits it's just me trying to get the build to work ha